openage/__init__.py

```diff
@@ -40,3 +42,14 @@
         compilerflags=config.COMPILERFLAGS,
         cython=config.CYTHONVERSION
     )
+
+# setup logging
+# do not overwrite any of the predefined levels
+# https://docs.python.org/3/library/logging.html#logging-levels
+logging.addLevelName(1, "MIN")
+logging.addLevelName(5, "SPAM")
+logging.addLevelName(51, "MAX")
+
+logging.setLoggerClass(CppLogger)
+logger = logging.getLogger('openage')
+logger.addHandler(CppHandler())
```

Comment:
Can we do the setup in the `log` python module?
So we could just import `log.setup` (or some better name) here and then call `log_setup()`, because then the logging code is kept in the right module (if the `logging.getLogger` needs to be passed, we can just give it to `log_setup` as argument).